contracts/Divvi.sol

```diff
@@ -12,17 +12,61 @@ contract DivviToken is
 {
   bytes32 public constant MINTER_ROLE = keccak256('MINTER_ROLE');
 
+  mapping(address => bool) private _hasTransferPermission;
+  bool private _permitTransfersOnly;
```

Comment:
The name `_permitTransfersOnly` suggests it's about permitting transfers, but when it's true, it actually makes transfers more restrictive.

It uses a double negative which can be hard to follow:
- when `_permitTransfersOnly` is `true`, only accounts with explicit permission can transfer
- when `_permitTransfersOnly` is false, everyone can transfer

Looks like we could name it `_restrictTransfers` or `_requireTransferPermission` instead?